### PR TITLE
remove deprecated reportdb schema image

### PR DESCRIPTION
**Pull request by mbussolotto**
# Description

Remove deprecated reportdb schema image. As soon the reportdb documentation will be created directly from the code, a link will be added here

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/specialized-guides/pages/large-deployments/hub-reporting.adoc [hunk 1]
```diff
@@ -62,8 +62,6 @@ On the {productname} Hub it is replaced with the real server id the managed serv
 
 Another common additional field is `synced_date`, which represent the time when the data were exported from the main {productname} Server database.
 
-image::reportdb_schema.png[]
```
Comment (mcalmer) [inline modules/specialized-guides/pages/large-deployments/hub-reporting.adoc:65]: Can we add some text where to find the documentation now?

**Review by mcalmer — CHANGES_REQUESTED** [review]

File: modules/specialized-guides/pages/large-deployments/hub-reporting.adoc [hunk 2]
```diff
@@ -62,8 +62,6 @@ On the {productname} Hub it is replaced with the real server id the managed serv
 
 Another common additional field is `synced_date`, which represent the time when the data were exported from the main {productname} Server database.
```
Comment (keichwa) [inline modules/specialized-guides/pages/large-deployments/hub-reporting.adoc:63]:
```suggestion
Another common additional field is `synced_date`, which represents the time when the data were exported from the main {productname} Server database.
```

**Review by keichwa — APPROVED** [review]

**Review by keichwa — APPROVED** [review]
If no more feedback arrives, or if nobody vetoes, I'll merge it tomorrow (or earlier if wanted).

**Comment by mbussolotto** [comment]
@mackdk do you think we are able to release the online documentation soon? or is there any link/path where to find the update schema documentation before it's online?  Thanks!

**Review by mcalmer — APPROVED** [review]

**Review by keichwa — APPROVED** [review]
Thanks, sounds good.  Contents ready to be merged, IMO.

Later we (doc-squad) can normalize it a little bit more according to the documentation style guide.

**Pull request merged**